scripts/combine_metadata.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+from augur.utils import read_metadata
+from Bio import SeqIO
+import csv
+
+EMPTY = ''
+
+# This script was written in preparation for a future augur where commands
+# may take multiple metadata files, thus making this script unnecessary!
+#
+# Merging logic:
+# - Order of supplied TSVs matters
+# - All columns are included (i.e. union of all columns present)
+# - The last non-empty value read (from different TSVs) is used. I.e. values are overwritten.
+# - Missing data is represented by an empty string
+#
+# We use one-hot encoding to specify which origin(s) a piece of metadata came from
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Custom script to combine metadata files",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument('--metadata', required=True, nargs='+', metavar="TSV", help="Metadata files")
+    parser.add_argument('--origins', required=True, nargs='+', metavar="STR", help="Names of origins (metadata columns will be created from these)")
+    parser.add_argument('--output', required=True, metavar="TSV", help="output (merged) metadata")
+    args = parser.parse_args()
+    return args
+
+if __name__ == '__main__':
+    args = parse_args()
+    assert(len(args.metadata)==len(args.origins))
+    assert(len(args.origins)>1)
```

Comment:
Could we convert these assertions to user-friendly error messages printed to stderr when either test fails?

Reply:
Yup! a16d89f